src/localization/en.ts

```diff
@@ -38,7 +38,9 @@ export const messages: { [key in MessageId]: string } = {
 
   'main.chat.ban': 'Ban chat',
   'main.chat.ban.confirmation': 'Are you sure you want to ban this user?',
-  'main.chat.navigation.banned': 'Banned chats',
+  'main.chat.navigation.banned': 'Banned',
+  'main.chat.unban': 'Restore chat',
+  'main.chat.unban.confirmation': 'Are you sure you want to restore this user?',
```

Comment:
Do you restore the user?

Reply:
Good question! This was the best translation I came up with. Ill gladly take any suggestions for a better one

Reply:
resurrect?

Reply:
:P

Reply:
This is under discussion in our mobile app channel